webapp/src/app/api/v0/cas/config.service.ts

```diff
@@ -0,0 +1,147 @@
+import crypto from "crypto";
+import { List, Map } from "immutable";
+import toml from "toml";
+
+import { Injectable } from "@angular/core";
+
+import { InstanceID } from "@dedis/cothority/byzcoin";
+
+export class Action {
+    constructor(
+        readonly darc: InstanceID,
+        readonly coin: InstanceID,
+    ) {}
+}
+
+const getField = (obj: object, name: string): unknown => {
```

Comment:
Why are these defined as `const` and not as `function`?

Reply:
Cruft from a previous iteration, fixing.